tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -224,102 +299,36 @@ async def test_train_persist_load_with_different_settings_non_windows(
             "intent_split_symbol": "+",
         },
         {"name": "CountVectorsFeaturizer"},
-        {"name": "DIETClassifier", MASKED_LM: True, EPOCHS: 1},
     ]
-    await _train_persist_load_with_different_settings(
-        pipeline, component_builder, tmp_path, should_finetune=False
-    )
-    await _train_persist_load_with_different_settings(
```

Comment:
seems we are dropping the `should_finetune=True` tests, aren't we?

Reply:
There weren't actually testing finetuning as they never trained again on the loaded model. There are tests lower in the file which actually testing finetuning properly.

Reply:
but they test if the classifier can be loaded correctly with the given config and in finetune mode, no?

Reply:
But nothing really different happens until you actually start training and `adjust_for_incremental_training` is called. And all of that is tested in other tests. 
I can add it back in, but I think it kind of gives a false expectation on what the test is actually doing.

Reply:
I've `parametrize`'d these tests to do it again.

Reply:
instead of parametrizing it - could we just try to load it twice? This would avoid one training and be great for the CI speed

Reply:
done